src/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -56,6 +56,7 @@ public string ClientVersion
         public LogCallback LoggingCallback { get; internal set; }
         public string Component { get; internal set; }
         public IDictionary<string, string> ExtraQueryParameters { get; internal set; } = new Dictionary<string, string>();
+        public bool UseNewDefaultRedirectUri { get; internal set; }
```

Comment:
"New" is an overloaded word. It might be "new" today, but 6 months from now, we will have forgotten all about the "old" obo:urn...

Reply:
UseRecommendedDefaultRedirectUri ?

Reply:
Shouldn't this just be UseDefaultRedirectUri?  This is now the default.  If we have a setting for using the old one (to force it for some reason) we should have UseOobRedirectUri or something./

Reply:
I believe we are still going to keep "urn:ietf:wg:oauth:2.0:oob" as the default redirect uri to avoid breaking customers which is why I named this "new" instead of just "UseDefaultRedirectUri".